### DISCUSSION TITLE
部署完之后发现所有下载的包都是http

### DISCUSSION THREAD

**Discussion by VisionBao** [tryzealot/zealot#788] [Q&A] [answered]
通过./deploy文件部署

我将自己的https证书拷贝到相应的目录中

发现访问网页的时候都是正常的https

而下载包，或者是下载描述文件的时候却是http，从而导致描述文件无法安装，ipa也无法安装到手机中。

尝试了几种方式也没有解决问题，请问应该如何排查这个问题？

**Comment by icyleaf** [comment, marked as answer]
发下反代的配置文件

**Reply by VisionBao** [reply]
没有使用反向代理，用的caddy

**Reply by icyleaf** [reply]
对于 zealot 来说 caddy 本身就是作为反代服务 😃

**Reply by VisionBao** [reply]
```
:443

tls /etc/caddy/certs/{$ZEALOT_CERT} /etc/caddy/certs/{$ZEALOT_CERT_KEY}
log
reverse_proxy zealot
~
```

**Reply by icyleaf** [reply]
找到问题了，caddy [2.5.0](https://github.com/caddyserver/caddy/releases/tag/v2.5.0) 版本做了反代的变更，需要兼容处理一下

**Reply by VisionBao** [reply]
caddy这个我并不常用，还需要进一步指点一下，是需要更改caddy的配置文件么？还是需要代码层级的修改？

**Reply by icyleaf** [reply]
我在代码层面做兼容吧，稍等下

**Reply by VisionBao** [reply]
好的，辛苦啦，那我这边等您兼容啦。

**Reply by icyleaf** [reply]
试试拉取最新的 `nightly` tag
